h3_test.go

```diff
@@ -106,6 +115,7 @@ func TestFromGeo(t *testing.T) {
 		Latitude:  67.194013596,
 		Longitude: 191.598258018,
 	}, 5)
+	t.Logf("Index:%x\n", h)
```

Comment:
bit of a nit, but can you remove this line, or alternatively, rephrase it inside `assert.Equal`'s message field, perhaps like:
```
assert.Equal(t, validH3Index, h, 
  "expected %x but got %x", 
  validH3Index, h)
```